core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/CourseDiscussionDetailViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.ustadmobile.core.viewmodel
+
+import com.ustadmobile.lib.db.entities.CourseDiscussion
+import com.ustadmobile.lib.db.entities.DiscussionPostWithDetails
+
+data class CourseDiscussionDetailUiState(
+    val courseDiscussion: CourseDiscussion? = null,
```

Comment:
I think as per call we don't need CourseDiscussion entity. Title, description, etc are on the courseblock.